app/src/main/java/com/aileyzhang/musicapp/Album.java

```diff
@@ -0,0 +1,28 @@
+package com.aileyzhang.musicapp;
+
+import android.graphics.Bitmap;
+
+/**
+ * Created by Ailey on 2017-12-22.
+ */
+
+public class Album {
+    public String mAlbumID;
+    public String mTitle;
+    public String mArtist;
+    public String mNumOfSongs;
```

Comment:
I would store this as an `int`, then format it accordingly wherever you display it (in this case, it's in the listview adapter for the album). This way this album data model class is more versatile, otherwise, you're bound to using this exact string wherever you use this class. :)